Sales.md

```diff
@@ -0,0 +1 @@
+guides/v2.2/mrg/ce/Sales.md
```

Comment:
seems like there is a testing symlink that should not be here

Reply:
Removed.